examples/exoskeleton/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "dependencies": {
+    "exoskeleton": "~0.3.0",
+    "backbone.localstorage": "git://github.com/yavorsky/Backbone.localStorage.git#a47fe5aa410f70dae9066c439d84483e4b32507e",
```

Comment:
@yavorsky Any chance we could switch to the official `backbone.localstorage` package?